ludwig/models/llm.py

```diff
@@ -142,6 +158,13 @@ def to_device(self, device):
                 self.model.save_pretrained(tmpdir)
                 self.model = AutoModelForCausalLM.from_pretrained(tmpdir, **model_kwargs)
 
+            # TODO(Arnav): Looking into this later, but it specifically happens when using local backend on a setup
```

Comment:
This happened because BERT didn't implement device_map, and therefore didn't load itself onto GPU when we asked it to.

Probably not an issue for real LLMs, but important to keep in mind when debugging in the future

Reply:
Totally agree